provision/aws/global-accelerator/accelerator_multi_az_create.sh

```diff
@@ -0,0 +1,127 @@
+      service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
+      service.beta.kubernetes.io/aws-load-balancer-healthcheck-path: "/lb-check"
+      service.beta.kubernetes.io/aws-load-balancer-healthcheck-protocol: "https"
+      service.beta.kubernetes.io/aws-load-balancer-healthcheck-interval: "10"
+      service.beta.kubernetes.io/aws-load-balancer-healthcheck-healthy-threshold: "3"
+      service.beta.kubernetes.io/aws-load-balancer-healthcheck-unhealthy-threshold: "3"
+  spec:
+    ports:
+    - name: https
+      port: 443
+      protocol: TCP
+      targetPort: 8443
+    selector:
+      app: keycloak
+      app.kubernetes.io/instance: keycloak
+      app.kubernetes.io/managed-by: keycloak-operator
+    sessionAffinity: None
+    type: LoadBalancer
+EOF
+  LB_DNS=$(waitForHostname ${SVC_NAME} ${NAMESPACE})
+  LB_ARN=$(aws elbv2 describe-load-balancers \
+    --query "LoadBalancers[?DNSName=='${LB_DNS}'].LoadBalancerArn" \
+    --region ${REGION} \
+    --output text
+  )
+  echo ${LB_ARN}
+}
+
+requiredEnv ACCELERATOR_NAME CLUSTER_1 CLUSTER_2 KEYCLOAK_NAMESPACE
+
+CLUSTER_1_REGION=$(rosa describe cluster -c ${CLUSTER_1} -o json | jq -r .region.id)
+CLUSTER_2_REGION=$(rosa describe cluster -c ${CLUSTER_2} -o json | jq -r .region.id)
+
+if [[ "${CLUSTER_1_REGION}" != "${CLUSTER_2_REGION}" ]]; then
+  echo "ROSA Clusters '${CLUSTER_1}' and '${CLUSTER_2}' must be deployed in the same AWS region for a Multi-AZ deployment"
+  exit 1
+fi
+
+ENDPOINT_GROUP_REGION=${CLUSTER_1_REGION}
+
+CLUSTER_1_ENDPOINT_ARN=$(createLoadBalancer ${CLUSTER_1} ${CLUSTER_1_REGION} ${ACCELERATOR_LB_NAME} ${KEYCLOAK_NAMESPACE})
+CLUSTER_2_ENDPOINT_ARN=$(createLoadBalancer ${CLUSTER_2} ${CLUSTER_2_REGION} ${ACCELERATOR_LB_NAME} ${KEYCLOAK_NAMESPACE})
+
+ACCELERATOR=$(aws globalaccelerator create-accelerator \
+  --name ${ACCELERATOR_NAME} \
+  --query 'Accelerator' \
+  --ip-address-type DUAL_STACK \
+  --output json
+)
+
+ACCELERATOR_ARN=$(echo ${ACCELERATOR} | jq -r .AcceleratorArn)
+ACCELERATOR_DNS=$(echo ${ACCELERATOR} | jq -r .DnsName)
+ACCELERATOR_DUAL_STACK_DNS=$(echo ${ACCELERATOR} | jq -r .DualStackDnsName)
+
+LISTENER_ARN=$(aws globalaccelerator create-listener \
+  --accelerator-arn ${ACCELERATOR_ARN} \
+  --port-ranges '[{"FromPort":443,"ToPort":443}]' \
+  --protocol TCP \
+  --query 'Listener.ListenerArn' \
+  --output text
+)
+
+ENDPOINTS=$(echo '
+[
+  {
+    "EndpointId": "'${CLUSTER_1_ENDPOINT_ARN}'",
+    "Weight": 50,
+    "ClientIPPreservationEnabled": false
+  },
+  {
+    "EndpointId": "'${CLUSTER_2_ENDPOINT_ARN}'",
+    "Weight": 50,
+    "ClientIPPreservationEnabled": false
+  }
+]' | jq -c .
+)
+
+ENDPOINT_GROUP_ARN=$(aws globalaccelerator create-endpoint-group \
+  --listener-arn ${LISTENER_ARN} \
+  --traffic-dial-percentage 100 \
+  --endpoint-configurations ${ENDPOINTS} \
+  --endpoint-group-region ${ENDPOINT_GROUP_REGION}
+)
+
+echo "ACCELERATOR DNS: ${ACCELERATOR_DNS}"
+echo "ACCELERATOR DUAL_STACK DNS: ${ACCELERATOR_DNS}"
```

Comment:
Do we need ipv6?
```suggestion
echo "ACCELERATOR DUAL_STACK DNS: ${ACCELERATOR_DUAL_STACK_DNS}"
```

Reply:
Good question! @ahus1 any thoughts?

Reply:
With IPv4 addresses costing money at AWS AFAIK as well as my local provider, and IPv6 is free, plus the world moving to IPv6, I suppose IPv6 is the thing.

Reply:
The configuration options are `IPV4` or `DUAL_STACK` so it's not possible to exclusively use IPv6. I'll leave it as `DUAL_STACK` for now.